third_party/nvfuser/python_tests/test_torchscript.py

```diff
@@ -801,7 +801,7 @@ def bool_not(x: torch.Tensor, y: torch.Tensor):
         jit_o = jitted(x, y)
         jit_o = jitted(x, y)
         o = bool_not(x, y)
-        self.assertEqual(o, jit_o)
+        self.assertEqual(o, jit_o)  # This test fails with eager constant ops
```

Comment:
With this PR, `jit_o` becomes filled with `True`. I'm not yet sure why, but I will look at our torchscript machinery to find what might be happening.